plenum/common/messages/node_messages.py

```diff
@@ -6,17 +6,20 @@
     MESSAGE_REQUEST, MESSAGE_RESPONSE
 from plenum.common.messages.fields import NonEmptyStringField, NonNegativeNumberField, IterableField, \
     SerializedValueField, SignatureField, TieAmongField, AnyValueField, RequestIdentifierField, TimestampField, \
-    LedgerIdField, MerkleRootField, Base58Field, LedgerInfoField, AnyField, ChooseField, AnyMapField
+    LedgerIdField, MerkleRootField, Base58Field, LedgerInfoField, AnyField, ChooseField, AnyMapField, \
+    LimitedLengthStringField
 from plenum.common.messages.message_base import MessageBase
 from plenum.common.types import f
 from plenum.common.messages.client_request import ClientMessageValidator
+from plenum.config import NAME_FIELD_LIMIT, DIGEST_FIELD_LIMIT, SENDER_CLIENT_FIELD_LIMIT, HASH_FIELD_LIMIT,\
+    SIGNATURE_FIELD_LIMIT, TIE_IDR_FIELD_LIMIT
 
 
 class Nomination(MessageBase):
     typename = NOMINATE
 
     schema = (
-        (f.NAME.nm, NonEmptyStringField()),
+        (f.NAME.nm, LimitedLengthStringField(max_length=NAME_FIELD_LIMIT)),
```

Comment:
It's not an issue but I think we can omit keyword for max_length here and in all such cases.

Reply:
if you don't mind, I would prefer to leave it as is